HBFA/UefiHostFuzzTestCasePkg/TestCase/MdeModulePkg/Library/SmmLockBoxLib/UpdateLockBoxTestCase/TestUpdateLockBoxFuzzOffset.c

```diff
@@ -82,7 +82,8 @@ RunTestHarness(
     0xdea652b0, 0xd588, 0x4c55, { 0xb5, 0xb4, 0xc6, 0x82, 0xe7, 0xa0,0x3d }
   };
 
-  SmmLockBoxSmmConstructor(NULL, NULL);
+  // commented due to no implementation

```

Comment:
Please do not modify the legacy test case

Reply:
fixed in [0148b9e](https://github.com/tianocore/edk2-staging/pull/386/commits/0148b9e1fc1d0947610f4bb1da22a6b373d9258e)